alert-common/src/main/java/com/synopsys/integration/alert/common/action/api/AbstractResourceActions.java

```diff
@@ -49,16 +54,17 @@ public AbstractResourceActions(DescriptorKey descriptorKey, ConfigContextEnum co
         this.context = context;
         // to do change the authorization manager to use the context enum and the descriptor key
         this.authorizationManager = authorizationManager;
-
     }
 
     protected abstract ActionResponse<T> createWithoutChecks(T resource);
 
     protected abstract ActionResponse<T> deleteWithoutChecks(Long id);
 
-    protected abstract ActionResponse<M> readAllWithoutChecks();
+    protected abstract List<D> getDatabaseModels();
```

Comment:
Can we use the word "retrieve" or "read" rather than "get"? The "get" method-prefix in Java is a ubiquitous convention for accessing _fields_ of a class, so it's nice to make a distinction.

Reply:
Do you want to apply the same level of precision to the accessors? I went off of their method convention of using 'get'.

Reply:
I'd argue that consistency is the better part of valor here - so if you object to 'get' as a prefix for non-trivial methods, we should alter the api of the accessors, or just leave the 'get' prefix alone.

Reply:
I was thinking about this with the accessors while modifying them recently and I agree that their prefixes should also change. We can lay one good brick with the actions right now and when the accessors aren't in such a high state of flux, we can refactor those as well.

Reply:
Well phrased. :)